sinks.go

```diff
@@ -65,6 +65,20 @@ func (p *Pipe) SHA256Sum() (string, error) {
 	return encodedCheckSum, nil
 }
 
+// Slice returns the contents of the pipe as a slice of strings, one element per line, or an error.
+// If there is an error reading the pipe, the pipe's error status
+// is also set.
```

Comment:
Looks like line wrapping went a bit wrong here—this third line would fit on the line above, wouldn't it?